src/components/rooms-tab.js

```diff
@@ -0,0 +1,153 @@
+    createdOn: new Date().toISOString(),
+    access: 'public',
+    owner: {
+      username: 'user1',
+      email: 'user1@test.com'
+    },
+    members: [{ userId: user._id, joinedOn: new Date().toISOString() }]
+  }, {
+    name: 'Room 2',
+    key: 'dsfdsdsf',
+    createdOn: new Date().toISOString(),
+    access: 'private',
+    owner: {
+      username: 'user2'
+    },
+    members: [{ userId: user._id, joinedOn: new Date().toISOString() }]
+  }
+];
+
+function RoomsTab() {
+  const user = useUser();
+  const { formatDate } = useDateFormat();
+  const { t } = useTranslation('roomsTab');
+
+  const renderName = (_value, room) => {
+    return <a href={fakeGetRoomUrl(room._id)}>{room.name}</a>;
+  };
+
+  const renderCreatedOn = (_value, room) => {
+    return <span>{formatDate(room.createdOn)}</span>;
+  };
+
+  const renderOwner = (_value, room) => {
+    return room.owner.email
+      ? <span>{room.owner.username} | <a href={`mailto:${room.owner.email}`}>{t('email')}</a></span>
+      : <span>{room.owner.username}</span>;
+  };
+
+  const renderJoinedOn = (_value, room) => {
+    const userAsMember = room.members.find(member => member.userId === user._id);
+    return <span>{formatDate(userAsMember.joinedOn)}</span>;
+  };
+
+  const renderAccess = (_value, room) => {
+    return <span>{t(`accessType_${room.access}`)}</span>;
+  };
+
+  const ownedRoomsColumns = [
+    {
+      title: t('name'),
+      dataIndex: 'name',
+      key: 'name',
+      render: renderName,
+      sorter: by(x => x.name)
+    },
+    {
+      title: t('common:createdOn'),
+      dataIndex: 'createdOn',
+      key: 'createdOn',
+      render: renderCreatedOn,
+      defaultSortOrder: 'descend',
+      sorter: by(x => x.createdOn),
+      width: '150px'
+    },
+    {
+      title: t('access'),
+      dataIndex: 'access',
+      key: 'access',
+      render: renderAccess,
+      sorter: by(x => x.access),
+      width: '150px'
+    }
+  ];
+
+  const membershipRoomsColumns = [
+    {
+      title: t('name'),
+      dataIndex: 'name',
+      key: 'name',
+      render: renderName,
+      sorter: by(x => x.name)
+    },
+    {
+      title: t('owner'),
+      dataIndex: 'owner',
+      key: 'owner',
+      render: renderOwner,
+      defaultSortOrder: 'descend',
+      sorter: by(x => x.owner),
+      width: '200px'
+    },
+    {
+      title: t('joinedOn'),
+      dataIndex: 'joinedOn',
+      key: 'joinedOn',
+      render: renderJoinedOn,
+      defaultSortOrder: 'descend',
+      sorter: by(x => x.joinedOn),
+      width: '150px'
+    },
+    {
+      title: t('access'),
+      dataIndex: 'access',
+      key: 'access',
+      render: renderAccess,
+      sorter: by(x => x.access),
+      width: '150px'
+    }
+  ];
+
+  return (
+    <div className="RoomsTab">
+      <section>
+        <h2 className="RoomsTab-sectionHeader">{t('ownedRoomsHeader')}</h2>
+        <Table dataSource={getOwnedRooms(user)} columns={ownedRoomsColumns} size="middle" />
```

Comment:
I know I've used tables in the project, but we should probably stay away from them when it comes to pages where the normal users operate, they don't really work on mobile. Should we discuss this and what to actually use?

Reply:
I think we can do a big table overhaul one day. Same goes for the search results. I wouldn't put too much thought into it right now, we don't know what else has to go on these pages yet.

Reply:
Agree, I wouldn't deviate from the pattern with this ticket. We already have the Search page which is all users facing and using a table.
We should revisit the layout choices independently so to do it consistently.